src/sentry/api/endpoints/event_details.py

```diff
@@ -44,25 +44,31 @@ def get(self, request, event_id):
         # HACK(dcramer): work around lack of unique sorting on datetime
         base_qs = Event.objects.filter(
             group=event.group_id,
-        ).exclude(id=event.id)
+        )
         try:
             next_event = sorted(
                 base_qs.filter(
-                    datetime__gte=event.datetime
+                    id__gt=event.id,
```

Comment:
If we can't do this filter (I don't see why we couldn't), I could just do a `filter()` in python.